VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -238,18 +245,44 @@ def post_to_wireserver(self, data):
 
                     http_util.connection.close()
                     if result.status != httplib.OK and result.status != httplib.ACCEPTED:
-                        raise Exception("encryption settings update request was not accepted")
+                        raise Exception("Encryption settings update request was not accepted")
```

Comment:
>update request [](start = 61, length = 14)

maybe change this exception message to not confuse with update_encryption_settings workflow itself. "Encryption settings could not be persisted". Or similar to Windows extension " Failed to send disk encryption data. Check KeyVault inputs, etc, "